src/network/protocol/messages/cmpctblock.zig

```diff
@@ -0,0 +1,201 @@
+const std = @import("std");
+const protocol = @import("../lib.zig");
+const Transaction = @import("../../../types/Transaction.zig");
```

Comment:
```suggestion
const Transaction = @import("../../../types/transaction.zig");
```

Reply:
https://github.com/supreme2580/btczee/blob/cmpctblock/src/types/transaction.zig

Reply:
done it